### PR TITLE
Split medal unlock requirements into seperate articles

### PR DESCRIPTION
**Pull request by TPGPL**
Continuation of #5702

### CODE REVIEW

**Comment by Walavouchey** [comment]
I can't really say whether the hush-hush medals should be listed with their requirements in the wiki, but having their flair texts at least would be interesting to keep for trivia/as  information for curious people

**Comment by peppy** [comment]
Yeah I don't think that should be a thing. I mean sure the information is available elsewhere, but if we are "officially" listing then what is the point of even calling them "hush-hush".

File: wiki/Medals/en.md [hunk 1]
```diff
@@ -156,142 +34,26 @@ All Mod Introduction medals are eligible to be unlocked in any [game mode](/wiki
 - The medals' brief descriptions, and
 - The *flair texts* <!-- https://twitter.com/ephemeralis/status/1068054191602638850 --> of the medals (if any), viewable by hovering over their corresponding cards.
```
Comment (Walavouchey) [inline wiki/Medals/en.md:35]: as per [asc](https://github.com/ppy/osu-wiki/blob/master/wiki/Article_styling_criteria/Formatting/en.md#comments)

```suggestion
- The *flair texts*<!-- https://twitter.com/ephemeralis/status/1068054191602638850 --> of the medals (if any), viewable by hovering over their corresponding cards.
```

File: wiki/Medals/Unlock_requirements/en.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Unlock requirements
+
+Index page for all articles regarding medal unlock requirements.
```
Comment (Walavouchey) [inline wiki/Medals/Unlock_requirements/en.md:3]: putting this here as mentioning would be nice, but perhaps optional.

```suggestion
Index page for all articles regarding medal unlock requirements. Hush-Hush medals are not included for obvious reasons.
```

**Review by Walavouchey — COMMENTED** [review]
looking good now 👍

**Comment by Ephemeralis** [comment]
Please don't list the hush-hush criteria on the official wiki. I know it is available on other sites, but half the fun (and in some cases, the difficulty) for those types of medals comes from attempting to decipher the hints and references in them.

**Comment by peppy** [comment]
This should also not be in git or github history so i'm going to delete this PR.

@TPGPL please rebase it out and fore push this branch.

**Comment by TPGPL** [comment]
Are you referring to [this commit](https://github.com/ppy/osu-wiki/pull/5736/commits/33797d7cabd2fd89ebaa30ca6b4793eeb067250a)? It removes the article with medal descriptions, not the actual solutions

**Comment by peppy** [comment]
Seems fine.

**Review by geshii — APPROVED** [review]
I don't see anything to change

**Pull request merged**